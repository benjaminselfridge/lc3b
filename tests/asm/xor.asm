        0x3400

        ;; Load R0 with 0xAAAA
        ADD     R0      R0      0xA
        LSHF    R0      R0      4
        ADD     R0      R0      0xA
        LSHF    R0      R0      4
        ADD     R0      R0      0xA
        LSHF    R0      R0      4
        ADD     R0      R0      0xA

        NOT     R1      R0

        XOR     R2      R1      -1

        TRAP    0x0
