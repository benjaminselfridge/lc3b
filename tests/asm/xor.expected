[ RegContains 0 0xAAAA,
  RegContains 1 0x5555,
  RegContains 2 0xAAAA
]